### ISSUE TITLE
Project status

### ISSUE THREAD

**Issue by bartvanhoutte** [async-interop/event-loop#156]
Hello everybody,

I've been doing quite a bit of asynchronous PHP lately, and we could really benefit from more interoperability between ReactPHP and Amp. It looks like a lot of work is being done in both ecospheres while it seems more logical to bundle that development time.

According to the homepage of this repository, the project should be considered a failure. I'm wondering what the reasons for that were and if they still apply today? Maybe it's time to give this another go?

Cheers

**Comment by kelunik** [comment]
Hey @bartvanhoutte,

ReactPHP and Amp _are_ interoperable via [`amphp/react-adapter`](https://github.com/amphp/react-adapter) now. As stated in the README, the API proposed here has been the base for Amp v2.x.

> I'm wondering what the reasons for that were and if they still apply today?

I guess the reasons were mostly two different approaches regarding the API and neither party being fine with the API of the other.

**Comment by bartvanhoutte** [comment]
Hey @kelunik,

I saw the adapter but it unless I'm mistaken it does not work the other way round?

Ok, sad to hear there wasn't enough common ground to continue.

**Comment by kelunik** [comment]
Yes, it doesn't, but if you're building an application, it shouldn't matter.
